tests/Integration/Queue/CustomPayloadTest.php

```diff
@@ -4,12 +4,15 @@
 
 use Illuminate\Contracts\Bus\QueueingDispatcher;
 use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Testing\TestCase;
 use Illuminate\Queue\Queue;
 use Illuminate\Support\ServiceProvider;
-use Orchestra\Testbench\TestCase;
+use Orchestra\Testbench\Concerns\CreatesApplication;
```

Comment:
This should be testing `Illuminate\Foundation\Testing\TestCase` and not `Orchestra\Testbench\TestCase`.

I would recommend adding similar tests on `orchestra/testbench-core` to cover this issue.

Reply:
`getPackageProviders` won't work after this change, right? I'll adjust it to manually register it

Reply:
@deleugpn `getPackageProviders` exists in `Orchestra\Testbench\Concerns\CreatesApplication`.

Reply:
For simple tests such as this, we don't need `Orchestra\Testbench\Concern\Testing` where all the setUp, tearDown, and all other feature customs to Testbench exists.